project/Dependencies.scala

```diff
@@ -14,6 +14,7 @@ object Dependencies {
   lazy val sbinary = "org.scala-tools.sbinary" %% "sbinary" % "0.4.2"
   lazy val sbtSerialization = "org.scala-sbt" %% "serialization" % "0.1.1"
   lazy val scalaCompiler = Def.setting { "org.scala-lang" % "scala-compiler" % scalaVersion.value }
+  lazy val scalaReflect = Def.setting { "org.scala-lang" % "scala-reflect" % scalaVersion.value }
```

Comment:
In other words, Pickling will pull in the version of scala-reflect that it used to build the release. I think I last used 2.10.4 thinking sbt would need it.

https://github.com/scala/pickling/blob/581f4b55b97eb7a200ca49661dcceaa356f4003e/build.sbt#L60-L61